src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(number1, number2) {
+      return (number1 + number2);
+    },
+    subtract(number1, number2) {
+      return (number1 - number2);
+    },
+    multiply(number1, number2) {
+      return (number1 * number2);
+    },
+    divide(number1, number2) {
```

Comment:
If the "number2" equals 0, the function should throw an error.

Reply:
Thanks! I appreciate your review and I'll fix the mistakes that you mentioned!